specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -900,6 +900,191 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/inventoryPolicies/{blobInventoryPolicyName}": {
+      "get": {
+        "tags": [
+          "BlobInventoryPolicies"
+        ],
+        "operationId": "BlobInventoryPolicies_Get",
+        "description": "Gets the blob inventory policy associated with the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountGetBlobInventoryPolicy": {
+            "$ref": "./examples/StorageAccountGetBlobInventoryPolicy.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/BlobInventoryPolicyName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Get the blob inventory policy successfully.",
+            "schema": {
+              "$ref": "#/definitions/BlobInventoryPolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "BlobInventoryPolicies"
+        ],
+        "operationId": "BlobInventoryPolicies_CreateOrUpdate",
+        "description": "Sets the blob inventory policy to the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountSetBlobInventoryPolicy": {
+            "$ref": "./examples/StorageAccountSetBlobInventoryPolicy.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
```

Comment:
Same comment as above (why not use the common parameter)?

Reply:
WIll send another PR, as this is the same pattern across all SRP APIs